src/middleware.ts

```diff
@@ -111,6 +111,15 @@ export function initializeMiddleware(app) {
   app.use(localsMiddleware)
   app.use(sameOriginMiddleware)
 
+  // Mount GraphQL proxy and cache
+  if (ENABLE_GRAPHQL_PROXY) {
+    app.use("/api/metaphysics", graphqlProxyMiddleware)
+  }
+
+  // Add CSRF to the cookie and remove it from the page. This allows the caching
+  // on the html and is used by the Login Modal to make secure requests.
+  app.use(csrfTokenMiddleware)
```

Comment:
Bingo @joeyAghion you were right regarding login 💥  Was a csrf middleware order of operations thing.